tests/test_acceptance.py

```diff
@@ -261,6 +266,20 @@ def test_req(request, slurm_system: SlurmSystem, partial_tr: partial[TestRun]) -
             ),
             SlurmContainerCommandGenStrategy,
         ),
+        "slurm_ray_container": lambda: create_test_run(
```

Comment:
Please also update fixture.params for this one, otherwise this case will not run.